components/PostNewsletter/PostNewsletter.jsx

```diff
@@ -19,7 +20,23 @@ const PostNewsletter = ({
   const { title, subtitle, img, imgAlt } = postNewsletterData
 
   const [isMessageHidden, setMessageHidden] = useState(true)
+  const [isMobile, setMobile] = useState(null)
   const withoutSubscribeForm = language !== 'ru' || type === 'news'
+
+  useEffect(() => {
+    const checkWindowWidth = () => {
+      if (window.matchMedia(defaultTheme.breakpoints.mobile.all.slice('@media '.length)).matches) {
+        setMobile(true) 
+      } else {
+        setMobile(false)
+      }
+    }
```

Comment:
Давай тут ещё вот так сделаем: 
```suggestion
    const checkWindowWidth = () => 
        setMobile(window.matchMedia(defaultTheme.breakpoints.mobile.all.slice('@media '.length)).matches) 
```